src/wp-includes/rest-api/endpoints/class-wp-rest-block-types-controller.php

```diff
@@ -512,6 +564,62 @@ public function get_item_schema() {
 					'context'     => array( 'embed', 'view', 'edit' ),
 					'readonly'    => true,
 				),
+				'variations'       => array(
+					'description' => __( 'Block variations.' ),
+					'type'        => 'array',
+					'items'       => array(
+						'type'       => 'object',
+						'properties' => array(
+							'name'        => array(
+								'description' => __( 'The unique and machine-readable name.' ),
+								'type'        => 'string',
+								'required'    => true,
+							),
+							'title'       => array(
+								'description' => __( 'A human-readable variation title.' ),
+								'type'        => 'string',
+								'required'    => true,
+							),
+							'description' => array(
+								'description' => __( 'A detailed variation description.' ),
+								'type'        => 'string',
+								'required'    => false,
+							),
+							'category'    => array(
+								'description' => __( 'Block type category classification, used in search interfaces to arrange block types by category.' ),
+								'type'        => 'string',
+								'required'    => false,
+							),
+							'icon'        => $icon_definition,
+							'isDefault'   => array(
+								'description' => __( 'Indicates whether the current variation is the default one.' ),
+								'type'        => 'boolean',
+								'required'    => false,
+								'default'     => false,
+							),
+							'attributes'  => array(
+								'description' => __( 'The attributes of the variation' ),
```

Comment:
This part could be further clarified as it might be confusing, it's not the schema but a way to initialize attributes with some initial values, maybe:
```suggestion
								'description' => __( 'The initial values for attributes.' ),
```

Reply:
Added in https://github.com/WordPress/wordpress-develop/pull/1015/commits/4fb70c610525ec7fee4d985a210d361a32ddf1b6